packages/app-bridge/src/types/Themes.ts

```diff
@@ -0,0 +1,67 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { LinkSettingsDisplay, LinkSettingsIconPosition } from './Document.ts';
+
+export interface CoverPage {
+    id(): number;
+    title(language?: string): string;
+    isPublished(): boolean;
+}
+
+export interface DocumentGroup {
+    id(): number;
+    name(language?: string): string;
```

Comment:
Why not `title`? It would make it consistent with all the other types

Reply:
As a DocumentGroup is something like a folder, it should be called `name`. We discussed this yesterday up and down and decided to keep it as is in the whole codebase already. Do you agree?

Reply:
Does the user of the interface care that it is a slightly different concept? Also, this doesn't seem like a good reason for giving a different name to what is essentially the same thing - the text you display as the entity's name/title/label. I prefer when things are consistent and predictable, so that I don't have to remember there is a special case where the `title` is called `name`. Just my two cents, feel free to do with it as you please ;-)

Reply:
As agreed with @oliverschwendener and @Kenny806 we’re going to change the property `name` to `title`.